src/topology/algebra/polynomial.lean

```diff
@@ -129,4 +131,78 @@ if hp0 : 0 < degree p
 then p.continuous.norm.exists_forall_le $ p.tendsto_norm_at_top hp0 tendsto_norm_cocompact_at_top
 else ⟨p.coeff 0, by rw [eq_C_of_degree_le_zero (le_of_not_gt hp0)]; simp⟩
 
+section roots
+
+open_locale polynomial
+open_locale nnreal
+
+variables {F K : Type*} [field F] [nontrivial K] [normed_field K]
```

Comment:
```suggestion
variables {F K : Type*} [field F] [normed_field K]
```
`normed_field` should imply `nontrivial`.

Reply:
Indeed. Thanks.